src/components/layout/__styles__/index.ts

```diff
@@ -6,6 +6,7 @@ import { css } from '@linaria/core'
 export const elHasGreyBackground = css``
 export const elHasMaxWidth = css``
 
+/** @deprecated */
```

Comment:
You can't deprecate ElMainContainer - needs to be removed because the React Component and the elMainContainer css class will both generate duplicate classes ie el-main-container